src/parts/Sidebar/SidebarContent/Navigation/index.tsx

```diff
@@ -169,7 +189,13 @@ const Navigation = ({
                 'h-6'
               )}
               aria-hidden='true' />
-            {t(navigationItem.name)}
+            {navigationItem.link === CROWDLOAN_DOMAIN ?
+            // TODO: not the nicest way of handling contextual navigation text, but
```

Comment:
This is out of scope for this ticket, but should be refactored if/when we need to extend context specific navigation